From 1660f86bef5d46cf84f7b6d5ad41d498cda63397 Mon Sep 17 00:00:00 2001
From: Piotr Gorski <lucjan.lucjanov@gmail.com>
Date: Sat, 21 May 2022 18:41:53 +0200
Subject: [PATCH 37/48] Revert "block, bfq: remove dead code for updating
 'rq_in_driver'"

This reverts commit 7fe559c11fe243ad8fc948b4aec4d10615021fe9.

Signed-off-by: Piotr Gorski <lucjan.lucjanov@gmail.com>
---
 block/bfq-iosched.c | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/block/bfq-iosched.c b/block/bfq-iosched.c
index 3b6067759..0d937f0fd 100644
--- a/block/bfq-iosched.c
+++ b/block/bfq-iosched.c
@@ -2320,6 +2320,22 @@ static sector_t get_sdist(sector_t last_pos, struct request *rq)
 	return 0;
 }
 
+#if 0 /* Still not clear if we can do without next two functions */
+static void bfq_activate_request(struct request_queue *q, struct request *rq)
+{
+	struct bfq_data *bfqd = q->elevator->elevator_data;
+
+	bfqd->rq_in_driver++;
+}
+
+static void bfq_deactivate_request(struct request_queue *q, struct request *rq)
+{
+	struct bfq_data *bfqd = q->elevator->elevator_data;
+
+	bfqd->rq_in_driver--;
+}
+#endif
+
 static void bfq_remove_request(struct request_queue *q,
 			       struct request *rq)
 {
-- 
2.36.1.74.g277cf0bc36

